### PR TITLE
Explained how to delete old & anonymize audit records

### PR DESCRIPTION
**Pull request by kgaikwad**

### CODE REVIEW

File: _includes/manuals/1.18/4.1.4_auditing.md [hunk 1]
```diff
@@ -15,3 +15,37 @@ parent object - so if a parameter was modified, you can see what host/group that
 
 
 Template changes also store a diff of the changes, and the ability to roll back to a previous version of the template.
+
+
```
Comment (ares) [inline _includes/manuals/1.18/4.1.4_auditing.md]: double empty line

File: _includes/manuals/1.18/4.1.4_auditing.md [hunk 2]
```diff
@@ -15,3 +15,37 @@ parent object - so if a parameter was modified, you can see what host/group that
...
+#### Expire old audits automatically
+
+
```
Comment (ares) [inline _includes/manuals/1.18/4.1.4_auditing.md]: double empty line

File: _includes/manuals/1.18/4.1.4_auditing.md [hunk 3]
```diff
@@ -15,3 +15,37 @@ parent object - so if a parameter was modified, you can see what host/group that
...
+You will probably want to delete your old audits after some time. To achieve this, we have a rake task.
+You can run task `foreman-rake audits:expire` manually whenever required.
+
+Available conditions:
```
Comment (ares) [inline _includes/manuals/1.18/4.1.4_auditing.md]: s/conditions/parameters/

File: _includes/manuals/1.18/4.1.4_auditing.md [hunk 4]
```diff
@@ -15,3 +15,37 @@ parent object - so if a parameter was modified, you can see what host/group that
...
+2. Expires all audits older then 7 days
+
+		<pre>foreman-rake audits:expire days=7</pre>
+
+
+Note that you can also set a cronjob which will delete your audits periodically using this rake task.
+
+#### Anonymize old audits
+
+You can anonymize your old audits instead of deleting the audit entries.
```
Comment (ares) [inline _includes/manuals/1.18/4.1.4_auditing.md]: I'd also add a sentence like "Anonymization clears links to user accounts and their IP addresses, but keeps all changes other audit data in database"

**Review by ares — CHANGES_REQUESTED** [review]
few nits in text, otherwise :+1:

**Comment by kgaikwad** [comment]
@ares,
Thank you for reviewing pull-request. I have updated the pull-request.

**Review by ares — APPROVED** [review]

**Comment by ares** [comment]
Content wise :+1: it would be good if someone could do the grammar check, but then it's good to go in. Thanks @kgaikwad

**Review by tbrisker — APPROVED** [review]
LGTM, Thanks @kgaikwad and @ares !

**Pull request merged**